token/token.go

```diff
@@ -56,6 +56,11 @@ func (mgm tokenManager) Extract(tokenString string) (*account.Identity, error) {
 		return nil, errors.New("Token not valid")
 	}
 
+	claimedUUID := token.Claims.(jwt.MapClaims)["uuid"]
+	if claimedUUID == nil {
+		return nil, errors.New("UUID can not be nil")
```

Comment:
It is better to use error object implemented the `error` interface: https://golang.org/pkg/builtin/#error